### PR TITLE
add RevokedBy field CoRE-2971

### PR DESCRIPTION
**Pull request by mmaxim**
@patrickxb @maxtaco 

I think this is all we need to do to get this info on `DeviceDetail` and `ComputedKeyInfo` (with some tests). Let me know though if I missed something.

### CODE REVIEW

**Comment by mmaxim** [comment]
CI is failing because my test user isn't checked in, I'll add a PR for that now on keybase/keybase

File: protocol/avdl/device.avdl [hunk 1]
```diff
@@ -16,6 +16,7 @@ protocol device {
     // not possible to determine revoker without major code change
```
Comment (cjb) [inline protocol/avdl/device.avdl]: Remove comment

Comment (mmaxim) [inline protocol/avdl/device.avdl]: Ugh I had removed that earlier, but reverted and redid the change and forgot to remove again. Sorry!

**Comment by patrickxb** [comment]
LGTM

**Pull request merged**